sentences/sv/_common.yaml

```diff
@@ -59,13 +59,15 @@ expansion_rules:
   alla: "(all | alla | allt)"
   ljusintensitet: "(ljusstyrka | intensitet | styrka | nivå | effekt)"
   ljuskällor: "(ljuskällor[na] | ljus[en | et] | [glöd|led|diod]lamp[ a | an | or | orna] | lysrör[en| et] | belysning[en])"
-  dimra: "(sätt | vrid | dra [på] | skruva [på] | dimra | dimma | ställ [in] | höj | sänk)"
+  dimra: "(ändra | sätt | vrid | dra [på] | skruva [på] | dimra | dimma | ställ [in] | höj | sänk)"
   gardiner: "([rull | motor]gardin[er | erna | en] | persienn[er | erna | en] | spjälgardin[er | erna | en] | draperi[er | erna | en]) | förhänge[n | na]"
   stäng_gardiner: "(stäng [igen | till]| dra (ner | igen | för | till ) | hissa ner | veckla (ner | igen) | tillslut) | slå (igen |ihop | för)"
   öppna_gardiner: "(öppna [upp] | dra (upp | bort | isär | undan) | hissa upp | veckla upp)"
   varmt_kallt: "(varmt | varm | kallt | kall | het | svalt )"
   temperature: "{temperature} [grader] [{temperature_unit}]"
+  change: "(färga | ändra | byt)"
```

Comment:
I think it’s better to call this byta_färg or similar as they are specific to change of colour and not generic changes.

Reply:
Agree, `färga` might also work as alias.

Reply:
I was under the impression that we where to use English variables names?

Reply:
@edgein666 I got an review comment about this in the beginning telling me that we should keep `expansion_rules` in Swedish unless the just wraps fields like `name` or 'area`
An right now most is in Swedish so we can try keep it like that and if we want to change it in the future it easy to replace all `expansion_rules`.

Reply:
Färga sounds good.